demos/utils.cc

```diff
@@ -21,13 +21,15 @@
 
 constexpr size_t kCacheLineSize = 64;
 
-// Forces a memory read of the byte at address p. This will result in the byte
-// being loaded into cache.
+// Forced memory load. Used during both real and speculative execution to create
+// a microarchitectural side effect in the cache. Also used for latency
+// measurement in the FLUSH+RELOAD technique.
 void ForceRead(const void *p) {
   (void)*reinterpret_cast<const volatile char *>(p);
```

Comment:
May add this deleted comment here?

// Forces a memory read of the byte at address p. This will result in the byte
// being loaded into cache.

I think the explicit explanation of what's happening in the code is useful here since the actual code is opaque.

The function comment is also good the way you changed it, but I think you should keep this code explanation too.

Reply:
OK, I'll add it in another PR.